x/market/client/cli/txDeal.go

```diff
@@ -150,57 +150,41 @@ func NewSellDataMsg(clientCtx client.Context, txf tx.Factory, fs *flag.FlagSet)
 		return txf, nil, fmt.Errorf("failed to parse data certificate file: %w", err)
 	}
 
-	encryptedDataUrlBytes, err := base64.StdEncoding.DecodeString(sellData.Cert.UnsignedCert.EncryptedDataUrlBase64)
-	if err != nil {
-		return txf, nil, err
-	}
-
-	dataHashBytes, err := base64.StdEncoding.DecodeString(sellData.Cert.UnsignedCert.DataHashBase64)
-	if err != nil {
-		return txf, nil, err
-	}
-
 	unSigned := types.UnsignedDataValidationCertificate{
-		DealId:               sellData.Cert.UnsignedCert.DealId,
-		DataHash:             dataHashBytes,
-		EncryptedDataUrl:     encryptedDataUrlBytes,
-		DataValidatorAddress: sellData.Cert.UnsignedCert.DataValidatorAddress,
-		RequesterAddress:     sellData.Cert.UnsignedCert.RequesterAddress,
-	}
-
-	signatureBytes, err := base64.StdEncoding.DecodeString(sellData.Cert.SignatureBase64)
-	if err != nil {
-		return txf, nil, err
+		DealId:               sellData.UnsignedCert.DealId,
+		DataHash:             sellData.UnsignedCert.DataHash,
+		EncryptedDataUrl:     sellData.UnsignedCert.EncryptedDataUrl,
+		DataValidatorAddress: sellData.UnsignedCert.DataValidatorAddress,
+		RequesterAddress:     sellData.UnsignedCert.RequesterAddress,
 	}
 
 	signed := types.DataValidationCertificate{
 		UnsignedCert: &unSigned,
```

Comment:
```suggestion
		UnsignedCert: sellData.UnsignedCert,
```
I think you can use this as it is.

Reply:
Ah, Please ignore this comment.